src/main/java/rx/Observable.java

```diff
@@ -39,7 +39,7 @@
  * <p>
  * For more information see the <a href="http://reactivex.io/documentation/observable.html">ReactiveX
  * documentation</a>.
- * 
+ *
```

Comment:
Okay, I'm going to the notepad and then I'm going to figure out how to turn these features off.